MagaluCloud/magalu
# feat(core): add prompt input executor

## Description
This PR implements everything we need for a new confirmation strategy on object-storage. When deleting a resource, instead of yes/no confirmation, the user need to retype the resource name 

## Related issues
- closes #837 

## How to test
Run` go run . object-storage objects delete --dst <bucket>/<key>`
You are expected to see a message like
```
This command will delete the object at <bucket-name>/<key>, and its result is NOT reversible. Confirm it by retyping the name of the bucket: <bucket-name>/<key>
```
If you type it wrongly you will see the message 
```
Error: You must type <bucket-name>/<key> to delete, aborted.
```

## Visual Reference
![Captura de tela de 2024-02-14 16-00-28](https://github.com/MagaluCloud/magalu/assets/110136151/093e0198-b194-42ff-a0d6-d39273b016e1)
